propagator/opentelemetry-propagator-aws-xray/src/opentelemetry/propagators/aws/aws_xray_propagator.py

```diff
@@ -71,6 +72,8 @@
 )
 
 TRACE_HEADER_KEY = "X-Amzn-Trace-Id"
+AWS_TRACE_HEADER_PROP = "com.amazonaws.xray.traceHeader"
```

Comment:
I don't think you need this one.  This was a java specific piece for looking it up in the java system properties, which is different from environment variables.

Reply:
thanks, removed